test/bdd/features/get.feature

```diff
@@ -0,0 +1,44 @@
+Feature: Get asset states test
+  Background: Setup local blockchain, bootstraps and nodes
+    Given the blockchain is set up
+    And 1 bootstrap is running
+
+  @release
+  Scenario: Get latest state of the knowledge asset
```

Comment:
I think that this is an error, we have the same test twice

Reply:
They're different, just forgot to change the name of the scenario. We're getting first and seconds states of the updated knowledge asset